### ISSUE TITLE
Built-in ForbiddenCode check (var_dump, shell_exec) is case-sensitive

### ISSUE THREAD

**Issue by weirdan** [vimeo/psalm#1547]
Psalm fails to report forbidden call to `var_dump()` when it's cased oddly.
https://psalm.dev/r/4e2bc2dc9c
```php
<?php
 vaR_dump("is not reported");
```
**Expected**: ForbiddenCode
**Actual**:
```
Psalm output (using commit 6e010d9): 

No issues!
```
